app/perfunwind.h

```diff
@@ -37,6 +37,7 @@
 #include <QString>
 #include <QMap>
 #include <QVariant>
+#include <QSet>
```

Comment:
unused, did you mean QHash?

Reply:
not really I used a QSet during testing and forgot to remove it
QHash get included somewhere else